ui/src/app/components/providers/config-renderer.tsx

```
@@ -222,68 +236,69 @@ export const ConfigRenderer: React.FC<{
                 updateParameter(param.key, v.toString())
               }
             />
-            {param.helpText && (
-              <p className="text-xs text-gray-500 mt-1">{param.helpText}</p>
-            )}
           </div>
         );
 
       case 'number':
+        const numberInputId = `num-${param.key}`;
```

Biome (2.5.5)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)